reana_workflow_engine_snakemake/executor.py

```diff
@@ -116,22 +120,6 @@ def run(
         # it would immediately check if the output files are present
         # and fail otherwise (3 sec timeout).
 
-        if job.is_norun:
```

Comment:
*question:* @mvidalgarcia do you remember if dealing with `all` rule is important? I deleted it and demo workflows are running fine, reports look the same for `master` and this PR.

Reply:
It's a [common practice in the Snakemake world ](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#targets-and-aggregation)to keep an `all` rule. We needed to handle it manually to `touch` the `jobfinished` file manually. I guess that with the new status polling this is not needed anymore.